packages/outputarea/src/widget.ts

```diff
@@ -105,9 +105,42 @@ export class OutputArea extends Widget {
     this.contentFactory =
       options.contentFactory || OutputArea.defaultContentFactory;
     this.layout = new PanelLayout();
-    for (let i = 0; i < model.length; i++) {
-      const output = model.get(i);
-      this._insertOutput(i, output);
+    const maximumTopBottomOutput = options.maximumTopBottomOutput || 10;
```

Comment:
What if `options.maximumTopBottomOutput === 0` disabled this feature so it is backward compatible?

Reply:
Great idea! will do